src/views/EventInitiation.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="page-container">
+    <NavBtns
+      :topType="'share-dark'"
+      :botType="currStep === 0 ? 'next-only' : 'default'"
+      @next="nextStep()"
+      @back="prevStep()"
+    />
+
+    <!-- step 1 -->
+    <div class="step-1 step" v-if="currStep === 0">
+      <img src="../assets/icons/event-initiation/hourglass-icon.svg" alt class="step-icon" />
+      <h1 class="step-title">Date / Time</h1>
+      <div class="date-wrapper">
+        <v-menu
+          ref="calendarMenu"
+          max-width="290"
+          :close-on-content-click="false"
+          offset-y
+          :return-value.sync="eventDate"
+        >
+          <template v-slot:activator="{ on }">
+            <textarea
+              type="text"
+              v-on="on"
+              placeholder="When should the event be held?"
+              class="step-input"
+              v-model="eventDate"
+            ></textarea>
+          </template>
+          <v-date-picker v-model="eventDate">
+            <v-btn text color="primary" @click="showCalendar = false">Cancel</v-btn>
+            <v-spacer></v-spacer>
+            <v-btn
+              color="primary"
+              @click="
+                $refs.calendarMenu.save(eventDate);
+                showDatePicker = false;
+                showTimePicker = true;
```

Comment:
I think this is supposed to bring up the timepicker modal but somehow it's not happening when I click 'save'. Instead the modal closes and on console I'd see error:
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Error in render: "RangeError: Invalid time value"

found in

---> <VDatePicker>
       <VThemeProvider>
         <VMenu>
           <EventInitiation> at src/views/EventInitiation.vue
             <VApp>
               <App> at src/App.vue
                 <Root>
warn @ vue.runtime.esm.js?2b0e:619
logError @ vue.runtime.esm.js?2b0e:1884
globalHandleError @ vue.runtime.esm.js?2b0e:1879
handleError @ vue.runtime.esm.js?2b0e:1839
Vue._render @ vue.runtime.esm.js?2b0e:3550
updateComponent @ vue.runtime.esm.js?2b0e:4066
get @ vue.runtime.esm.js?2b0e:4479
run @ vue.runtime.esm.js?2b0e:4554
flushSchedulerQueue @ vue.runtime.esm.js?2b0e:4310
eval @ vue.runtime.esm.js?2b0e:1980
flushCallbacks @ vue.runtime.esm.js?2b0e:1906
Promise.then (async)
timerFunc @ vue.runtime.esm.js?2b0e:1933
nextTick @ vue.runtime.esm.js?2b0e:1990
queueWatcher @ vue.runtime.esm.js?2b0e:4402
update @ vue.runtime.esm.js?2b0e:4544
notify @ vue.runtime.esm.js?2b0e:730
reactiveSetter @ vue.runtime.esm.js?2b0e:1055
proxySetter @ vue.runtime.esm.js?2b0e:4631
input @ EventInitiation.vue?cf6d:86
invokeWithErrorHandling @ vue.runtime.esm.js?2b0e:1854
invoker @ vue.runtime.esm.js?2b0e:2179
original._wrapper @ vue.runtime.esm.js?2b0e:6917
vue.runtime.esm.js?2b0e:1888 RangeError: Invalid time value
    at Object.eval [as titleDate] (createNativeLocaleFormatter.ts?dfaa:32)
    at VueComponent.genPickerTitle (VDatePicker.ts?9174:340)
    at VueComponent.genPicker (index.ts?99ca:49)
    at Proxy.render (VDatePicker.ts?9174:483)
    at VueComponent.Vue._render (vue.runtime.esm.js?2b0e:3548)
    at VueComponent.updateComponent (vue.runtime.esm.js?2b0e:4066)
    at Watcher.get (vue.runtime.esm.js?2b0e:4479)
    at Watcher.run (vue.runtime.esm.js?2b0e:4554)
    at flushSchedulerQueue (vue.runtime.esm.js?2b0e:4310)
    at Array.eval (vue.runtime.esm.js?2b0e:1980)
```

Reply:
timepicker isn't working i'm still trying to solve it, but right now it doesn't break, although i can comment out the timepicker stuff

Reply:
Still getting errors thoooo so should probably entirely remove first (and start another issue to do this) or implement fully 😢

Reply:
yep working on that right now

Reply:
mate go back to actual paid work 🤣HAHAHAHHA